roles/os10_route_map/templates/os10_route_map.j2

```diff
@@ -184,6 +184,13 @@ route-map {{ vars.name }} permit 10
           {% endif %}
           {% if vars.match is defined and vars.match %}
             {% for match in vars.match %}
+              {% if match.as_path is defined %}
+                {% if match.as_path %}
+ match as-path {{ match.as_path }}
+                {% else %}
+ no match as-path {{ match.as_path }}
```

Comment:
I think you are proposing a redesign of the whole `match` structure, which is out of scope for this pull request

Reply:
Thank you for considering the review comment. Can you please verify with the yaml structure of single match stmt instead of list of match values. I think match as-path is not supported as a list in implementation. And also with yaml change alone, template changes(.j2 file) would work fine.

Reply:
@Appal5 I am saying, that `match` is set in the documentation as a list.

Please see

https://github.com/ansible-collections/dellemc.os10/pull/75/files#diff-18141c27c903d62cb23e4bf9e4df86d781a2d9cef0b8d5ea559c49f794e07bc0L45

If you are requesting that I change it from a list to a single item, that is a very large change that is out of scope for this pull request and I am not going to make that change.

Reply:
Yeah. Got your point. But yaml format was misleading when we see it separately for as-path attribute. You can go ahead with this PR. Thanks

Reply:
@Appal5  The reason its a list, since apart from as-path other match items can be a list. as-path was not there when initially the role was designed.  
@sc68cal The changes are fine.